spec/frequency_table_spec.rb

```diff
@@ -193,42 +193,36 @@
       end
 
       context "with Serials" do
+        # umich deposited one htitem,
```

Comment:
None of the logic of this test has changed, but the use of fake organizations made it hard to read and apparently brittle.